packages/web/src/components/reposition/reposition-info/RepositionInfo.tsx

```diff
@@ -79,8 +79,7 @@ const RepositionInfo: React.FC<RepositionInfoProps> = ({
               placement="top"
               FloatingContent={
                 <ToolTipContentWrapper>
-                  The deposit ratio of the two tokens is determined based on the
-                  current price and the set price range.
+                  The multiplier calculated based on the concentration of your range. This indicates how much more rewards you can earn compared to a full range position with the same capital.
```

Comment:
What about managing it as a constants variable?

Reply:
I'm going to refactor it later